examples/Diffusion/README.md

```diff
@@ -1 +1,122 @@
-The Diffusion solver was written by Chris Cantwell and Dave Moxey as a Nektar++ app for the NEPTUNE project. The original source code and accompanying documentation can be found in the [nektar-diffusion](https://github.com/ExCALIBUR-NEPTUNE/nektar-diffusion) GitHub repository.
+# The Diffusion solver
+
+## Overview
+The Diffusion solver was written by Chris Cantwell and Dave Moxey as a Nektar++ proxyapp for the NEPTUNE project.
```

Comment:
For these solver attributions are we sure there are not any postdocs/phd students that we should recognise?

Reply:
Good point; will check with Dave

Reply:
Just remembered I asked Dave about this for another purpose; he suggested adding Jacques as a coauthor of nektar-driftwave. Have updated the text accordingly.